packages/2018-transportation-systems/src/state/decline-in-ridership/actions.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+
+import importAdapter from '../import-adapter';
+import actionEmitter from '../import-adapter-action-emitter';
+
+export const IMPORT_START = 'DECLINE_IN_RIDERSHIP_OVER_TIME/START';
+export const IMPORT_SUCCESS = 'DECLINE_IN_RIDERSHIP_OVER_TIME/SUCCESS';
+
+export const ridershipOverTimeStart = actionEmitter(IMPORT_START);
+export const ridershipOverTimeSuccess = actionEmitter(IMPORT_SUCCESS);
+
+const DECLINE_IN_RIDERSHIP_API = 'http://service.civicpdx.org/transportation-systems/passenger-census/system/annual/total/?format=json';
+
+const importPromise = axios.get(DECLINE_IN_RIDERSHIP_API);
```

Comment:
It's great that the `importAdapter` still works here since it just takes a promise, but I think it would be better to make an `apiAdapter` that is just like the `importAdapter` but takes a URL.

This way `axios` becomes an implementation detail that not every duck has to be concerned with.